change/@ni-nimble-components-e1eca008-24b5-481f-a835-1e3157d4ac1b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
> it's showing a diff in the mention view story because the mention is now rendering as black instead of green.

Regarding this change, it is not because of the `disable-editing`, we have added a new CSS variable to control the color of mention view element when the editor gets disabled. Please take a look at the discussion [here ](https://github.com/ni/nimble/pull/1683#discussion_r1403301987). As of this change, the color will be set for the CSS variable only in the editor/viewer. So this is looking black in a standalone case but green when used along with the editor/viewer.

And for the change of mention view from bold to plain body font, We thought of updating it in the list box PR, But there is a decoration tag here as well right, So I have updated that font in this PR itself and updated the test.

> There are no diffs in the editor matrix tests which means we're not doing visual testing of a lot of this new work.

Updated this matrix test with the same markdown content as of viewer.

Reply:
The matrix test still needs to be updated to increase the height of the editors. Right now you can't see all the text so the diff isn't exercising the mention view.

Reply:
Oops, I didn't see your latest comment when I edited the one above. Glad you were able to get it working.